src/components/generic/InputField/InputField.jsx

```diff
@@ -16,45 +16,47 @@ const HINTS = {
  * Input Field component.
```

Comment:
Generic input field component for building more specific components (text, date, number, etc.). Should not be used directly in forms.